projects/cincinnati/cincinnati.yaml

```diff
@@ -1,32 +0,0 @@
-apiVersion: v1
-kind: List
-items:
-- kind: ImageStream
-  apiVersion: v1
-  metadata:
-    name: cincinnati-build-root
-- kind: BuildConfig
-  apiVersion: v1
-  metadata:
-    name: cincinnati-build-root-latest
-    labels:
-      ci.openshift.io/source-repo: cincinnati
-  spec:
-    successfulBuildsHistoryLimit: 1
-    failedBuildsHistoryLimit: 2
-    output:
-      to:
-        kind: ImageStreamTag
-        name: cincinnati-build-root:latest
```

Comment:
Is it still needed?

https://github.com/openshift/release/blob/93f891aac202d6af2a336fbcbf0691a101e53716/ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml#L10-L14

I think we need to move this file, together with https://github.com/openshift/release/tree/master/services/cincinnati-ci, to `app.ci`'s folder.

Reply:
It seems that their postsubmits are building the image and promote it to that imagestream. There is no need to have a buildconfig for that. see https://github.com/openshift/release/tree/master/services/cincinnati-ci and https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml#L15-L36